client/containers/Pub/PubHeader/collections/CollectionBrowser.tsx

```diff
@@ -82,6 +88,15 @@ const CollectionBrowser = (props: Props) => {
 						}
 					/>
 				))}
+			{isLoading && (
+				<MenuItem
+					disabled
+					className="loading-menu-item"
+					textClassName="menu-item-text"
+					icon={<Spinner size={30} />}
+					text="Loading..."
+				/>
+			)}
```

Comment:
i moved the spinner down, as loading and showing pubs aren't mutually exclusive anymore.